File: fabric/src/main/java/juuxel/adorn/compat/TraverseCompat.java
package juuxel.adorn.compat;

import juuxel.adorn.block.variant.BlockVariant;
import juuxel.adorn.block.variant.CompatBlockVariantSet;

import java.util.List;

public final class TraverseCompat implements CompatBlockVariantSet {
    @Override
    public String getModId() {
        return "traverse";
    }

    @Override
    public List<BlockVariant> getWoodVariants() {
        return List.of(new BlockVariant.Wood("traverse/fir"));
    }
}

File: fabric/src/main/kotlin/juuxel/adorn/compat/TraverseCompat.kt
package juuxel.adorn.compat

import juuxel.adorn.api.block.BlockVariant
import juuxel.adorn.block.BlockVariantSet

object TraverseCompat : BlockVariantSet {
    override val woodVariants = listOf(BlockVariant.Wood("traverse/fir"))
}
